src/Datagrid/SimplePager.php

```diff
@@ -86,8 +86,41 @@ public function countResults(): int
         return $n;
     }
 
+    public function getCurrentPageResults(): iterable
+    {
+        if ($this->results) {
```

Comment:
If `$this->results` is an array with 0 elements, this condition will not be met. Is that what we want here?